osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSlider.cs

```diff
@@ -28,6 +29,7 @@ public class DrawableSlider : DrawableOsuHitObject, IDrawableHitObjectWithProxie
 
         public readonly SnakingSliderBody Body;
         public readonly SliderBall Ball;
+        private readonly Container<DrawableSliderTick> ticks;
```

Comment:
can you just use linq `OfType` rather than storing this?

Reply:
This is actually the container being added to the children of `DrawableSlider`. I'm using it for the sake of convenience and removing complexity.